tests/dotnet-sos/hello/hello.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>netcoreapp6.0</TargetFramework>

```

Comment:
Should we use net6.0?

Reply:
Both net6.0 and netcoreapp6.0 seem to work.